src/directives/formly-form.test.js

```diff
@@ -509,8 +526,9 @@ describe('formly-form', () => {
       }
 
       scope.$digest()
+      $timeout.flush()
```

Comment:
Nope, `$digest` is needed to trigger the watcher on formly directive for the `model` property and `$timeout` is just a dirty workaround for `runExpressions` method :)